### PR TITLE
Remove empty element from keybinding label array

### LINKED ISSUE #91235: "+" is concatenated at the end of key combination in screencast mode
Issue Type: <b>Bug</b>

When using screencast mode, if you press a sequence of keys such as "Ctrl" + "Alt", Visual Studio Code adds an extra + at the end of the sequence.

image

VS Code version: Code 1.42.1 (c47d83b293181d9be64f27ff093689e8e7aed054, 2020-02-11T14:45:59.656Z)
OS version: Windows_NT x64 10.0.18362

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz (4 x 2712)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off<br>surface_control: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.93GB (1.05GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (32)</summary>

Extension|Author (truncated)|Version
---|---|---
30-seconds-of-code|30-|1.0.0
better-comments|aar|2.0.5
Bookmarks|ale|11.0.0
vscode-color|ans|0.4.5
bracket-pair-colorizer|Coe|1.0.61
javascript-ejs-support|Dig|1.3.1
vscode-npm-script|eg2|0.3.11
vsc-material-theme|Equ|32.5.0
vsc-material-theme-icons|equ|1.1.4
php-intellisense|fel|2.3.14
vscode-mocha-test-adapter|hbe|2.2.2
vscode-test-explorer|hbe|2.17.0
python|ms-|2020.2.64397
remote-wsl|ms-|0.42.3
vsliveshare|ms-|1.0.1653
material-icon-theme|PKi|3.9.3
java|red|0.57.0
vscode-xml|red|0.10.1
code-settings-sync|Sha|3.4.3
vscode-spotify|shy|3.1.0
java-generate-setters-getters|soh|5.3.0
code-spell-checker|str|1.7.24
vscodeintellicode|Vis|1.2.5
vscode-java-debug|vsc|0.25.0
vscode-java-dependency|vsc|0.9.0
vscode-java-pack|vsc|0.8.1
vscode-java-test|vsc|0.22.1
vscode-maven|vsc|0.21.0
change-case|wma|1.0.0
vscode-wpilib|wpi|2020.2.2
viml|Xad|0.0.2
material-theme|zhu|3.2.3

(1 theme extensions excluded)

</details>

### PR DESCRIPTION
**Pull request by ghost**
Fixes #91235 

vscode

### CODE REVIEW

File: src/vs/workbench/services/keybinding/test/electron-browser/macLinuxFallbackKeyboardMapper.test.ts [hunk 1]
```diff
@@ -228,10 +228,10 @@ suite('keyboardMapper - LINUX fallback', () => {
 				code: null!
 			},
 			{
-				label: 'Ctrl+',
+				label: 'Ctrl',
 				ariaLabel: 'Control+',
```
Comment (jaqra) [inline src/vs/workbench/services/keybinding/test/electron-browser/macLinuxFallbackKeyboardMapper.test.ts]:
```suggestion
				ariaLabel: 'Control',
```

File: src/vs/workbench/services/keybinding/test/electron-browser/macLinuxKeyboardMapper.test.ts [hunk 2]
```diff
@@ -780,18 +780,18 @@ suite('keyboardMapper - LINUX de_ch', () => {
 				code: 'ControlLeft'
 			},
 			{
-				label: 'Ctrl+',
+				label: 'Ctrl',
 				ariaLabel: 'Control+',
```
Comment (jaqra) [inline src/vs/workbench/services/keybinding/test/electron-browser/macLinuxKeyboardMapper.test.ts]:
```suggestion
				ariaLabel: 'Control',
```

File: src/vs/workbench/services/keybinding/test/electron-browser/macLinuxKeyboardMapper.test.ts [hunk 3]
```diff
@@ -1204,10 +1204,10 @@ suite('keyboardMapper - LINUX en_us', () => {
 				code: 'ControlRight'
 			},
 			{
-				label: 'Ctrl+',
+				label: 'Ctrl',
 				ariaLabel: 'Control+',
```
Comment (jaqra) [inline src/vs/workbench/services/keybinding/test/electron-browser/macLinuxKeyboardMapper.test.ts]:
```suggestion
				ariaLabel: 'Control',
```

**Comment by joaomoreno** [comment]
@jaqra Let's not fix this that deep down and keep the fixes in the screencast mode... even if you do some string magic. Wanna submit another PR with a simper fix?

**Comment by jaqra** [comment]
@joaomoreno yes im gonna send another pr :)

**Pull request closed without merging**